packages/bot-web-ui/src/components/load-modal.jsx

```diff
@@ -232,6 +232,7 @@ const LoadModal = ({
         toggleModal={toggleLoadModal}
         onMount={onMount}
         onUnmount={onUnmount}
+        dom_to_ignore={document.querySelectorAll('.injectionDiv')[0]}
```

Comment:
- Could we rename it to `elements_to_ignore` and pass an array of elements that should be ignored? This may be useful when we want to ignore multiple elements in the future.
- You may use `document.querySelector()` if you only want the first element returned by the selector.

```suggestion
        elements_to_ignore={[document.querySelector('.injectionDiv')]}
```

Reply:
Sure 👍